Dockerfile

```diff
@@ -124,3 +124,13 @@ RUN : \
     && tar --strip-components 1 --directory /opt/swift -xf /tmp/swift.tar.gz \
     && rm /tmp/swift.tar.gz \
     && :
+
+RUN echo 'end: minimal'
+ENV PATH=/opt/r/:$PATH
+RUN : \
+    echo 'lang: r' \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends r-base \
```

Comment:
this will install things outside of `/opt/r` so this will not work -- is there a way to get a full installation entirely in `/opt/r` ?

Reply:
From the PR description:

> Installing R into a specific location is only possible when building from source, and that requires additional build dependencies like Fortran Compiler, Make etc. I assumed it's enough to locate the execution binary in /opt/r, so I just moved it there after installation.

Reply:
It might also be possible otherwise though, I am not an expert. I concluded this from https://unix.stackexchange.com/questions/149451/install-r-in-my-own-directory.

Reply:
Here's an instruction from a good source on how to install R from source: https://docs.rstudio.com/resources/install-r-source/

Reply:
So are you saying moving the executable into `/opt/r` is not enough and we must go for the source installation if there is no other way of locating a binary installation in `/opt/r`?

Reply:
correct, this layer must have only a single root in `/opt` and no files elsewhere